gdx/src/com/badlogic/gdx/graphics/glutils/FrameBufferCubemap.java

```diff
@@ -64,6 +64,9 @@
 	/** the zero-based index of the active side **/
 	private int currentSide;
 
+	/** cubemap sides cache */
+	private Cubemap.CubemapSide[] cubemapSides = Cubemap.CubemapSide.values();
```

Comment:
If you make this `private static final`, you don't even have the per-instance overhead.